test/integration/test-cases/chart-needs.sh

```diff
@@ -2,6 +2,14 @@ if [[ helm_major_version -eq 3 ]]; then
   chart_need_case_input_dir="${cases_dir}/chart-needs/input"
   chart_need_case_output_dir="${cases_dir}/chart-needs/output"
 
+  config_file="helmfile.yaml"
+  if [[ ${HELMFILE_V1MODE} = true ]]; then
+    pushd "${chart_need_case_input_dir}"
+    mv "${config_file}" "${config_file}.gotmpl"
+    config_file="${config_file}.gotmpl"
+    popd
+  fi
```

Comment:
This is a new idiom I used in all the integration test runner script to rename helmfile.yaml to helmfile.yaml.gotmpl only when it's run with v1 mode.